tox.ini

```diff
@@ -131,4 +131,5 @@ deps = coverage
 skip_install = true
 commands=
    coverage combine
+   coverage xml
```

Comment:
This is needed to write out a `coverage.xml` file to the root directory. The reason is that the combined `.coverage-reports/.coverage` file can't be picked up by the `codecov` program because of https://github.com/codecov/codecov-python/issues/198.